ex10/index.js

```diff
@@ -0,0 +1,11 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert a integer number:");
+const num = Number(prompt(">"));
+
+function repeat() {
+    for (let i = 0; i < 10; i++)
+        console.log(num);
+}
+
+console.log(repeat());
```

Comment:
Why are you calling `console.log` on a function that does not return anything?

Reply:
I hadn't thought about that, fixed.